src/TextToTalk/TextToTalk.cs

```diff
@@ -419,6 +426,31 @@ private string GetSpeakerRace(GameObject? speaker)
             return row.Masculine;
         }
 
+        private unsafe string GetSpeakerAge(GameObject? speaker)
+        {
+            if (speaker is null || speaker.Address == nint.Zero)
+            {
+                return "Unknown";
+            }
+
+            var charaStruct = (FFXIVClientStructs.FFXIV.Client.Game.Character.Character*)speaker.Address;
+            var speakerBodyType = charaStruct->DrawData.CustomizeData.BodyType;
+
+            switch (speakerBodyType)
+            {
+                case 0:
+                    return "Unknown";
+                case 1:
+                    return "Adult";
+                case 3:
+                    return "Elder";
+                case 4:
+                    return "Youth";
+                default:
+                    return "Unknown";
+            }
```

Comment:
If these aren't coming from a sheet, I'd prefer this being implemented as a `BodyType` enum for the raw data and storing that in `SayRequest`, and then sending this over the wire as a string - Mapperly can do the conversion.